packages/component-library/components/OffCanvas/components/Menu.tsx

```diff
@@ -1,34 +1,34 @@
 import classNames from "classnames"
 import * as React from "react"
+import { MenuProps } from "../../NavigationBar/types"
 
 const styles = require("./Menu.module.scss")
 
 type Props = {
-  links: any[]
+  section: string
+  links: MenuProps
 }
 
-const Menu = ({ links }: Props) => {
-  const indexOfFirstSecondaryLink = links.findIndex(
-    link => link.props.secondary
+const Menu = ({ links, section }: Props) => {
+  const renderNavItem = (link: MenuProps, index) => (
+    <li
+      key={index}
```

Comment:
I know this was part of the package before, but index as keys can cause unnecessary re-renders, I see that the `uuid` package is a dependency. Could we use this + an 'almost-unique' sting to render a key.

E.G. the `url` for links, and `heading` for menu items

Reply:
Good pickup! I'll change those up